.github/e2e/docker-compose.yml

```diff
@@ -1,21 +1,20 @@
 services:
-  rust-peer:
-    image: ${RUST_PEER_IMAGE:-fluencelabs/nox:unstable_minimal}
+  nox:
+    image: ${NOX_IMAGE:-fluencelabs/nox:unstable_minimal}
     ports:
       - 7771:7771
       - 9991:9991
     environment:
-      - FLUENCE_ENV_AQUA_IPFS_EXTERNAL_API_MULTIADDR=/dns4/ipfs/tcp/5001
+      - FLUENCE_ENV_AQUA_IPFS_EXTERNAL_API_MULTIADDR=/dns/localhost/tcp/5001
```

Comment:
don't use localhost plz, it may become wild things like ipv6

Reply:
@nahsi so what do we need to use for the external address?

Reply:
127.0.0.1